Dockerfile

```
@@ -101,10 +101,13 @@ RUN set -eux; \
 FROM app_php AS app_php_dev
 
-RUN apk add --no-cache bash
-RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.alpine.sh' | bash
-RUN apk add symfony-cli
-RUN apk add --no-cache make
+RUN apk add --no-cache \
+    bash \
+    make \
+    bats
+
+RUN curl -sS https://get.symfony.com/cli/installer | bash \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)